src/entity/effect/VanillaEffects.php

```diff
@@ -68,6 +69,7 @@ protected static function setup() : void{
 		//TODO: bad_omen
 		self::register("blindness", new Effect(KnownTranslationFactory::potion_blindness(), new Color(0x1f, 0x1f, 0x23), true));
 		self::register("conduit_power", new Effect(KnownTranslationFactory::potion_conduitPower(), new Color(0x1d, 0xc2, 0xd1)));
+		self::register("darkness", new Effect("%potion.darkness", new Color(0x29, 0x27, 0x21), true));
```

Comment:
Don't hardcode this. Use the KnownTranslations API. I know it doesn't exist yet, but that will ensure the build is red until the translation has been added, so that no one mistakenly merges the PR.

Reply:
Actually, it appears this already exists: https://github.com/pmmp/PocketMine-MP/blob/069062f122f5aa1d1a97aaebd8ea896c0ceca490/src/lang/KnownTranslationFactory.php#L746

Reply:
It is weird that it is called `effect_darkness`, but I'll change it to that